### PR TITLE
ZCS-1451: notes on flushing cache after adding imap node

### PR DESCRIPTION
**Pull request by ilyusha**
Added a paragraph about flushing the LDAP cache after adding an imapd node.

### CODE REVIEW

File: imapd.adoc [hunk 1]
```diff
@@ -11,3 +11,5 @@ Whenever a new IMAPD node is installed in a ZCS cluster that is not a mailboxd i
 throttle the new server.  Failure to do so will cause SOAP traffic from the Remote IMAPD node to be throttled leading to unexpected communication errors.
 
 Alternatively the DoSFilter can be effectively disabled by adding the ip address subnet that the new machine lives on to the '''zimbraHttpThrottleSafeIPs''' configuration item.
+
+Lastly, when an IMAPD node is added to a ZCS cluster, the globalconfig LDAP cache must be flushed on all ZCS servers that are running a proxy. This is necessary to ensure that new node is added to '''zimbraReverseProxyUpstreamImapServers''' attribute; without this step, the proxy will not be aware of the newly-provisioned server. This can be done via command line by running `zmprov fc -a config`. To verify that this has taken effect, make sure that the new IMAPD node is listed in the output of `zmprov gacf zimbraReverseProxyUpstreamImapServers`, when run from a proxy node.
```
Comment (fatkudu) [inline imapd.adoc]: Suggest adding a "the" i.e.
s/ensure that new node is added/ensure that **the** new node is added/

Comment (ilyusha) [inline imapd.adoc]: Ah, good catch.

Comment (tmclane) [inline imapd.adoc]: I should add similar language to the 'install guide' where I added information regarding the installation of a remote imapd server correct?

**Comment by gordyt** [comment]
I'm seeing some warnings when generating the output that may need to be addressed:

    asciidoctor: WARNING: backuprestore.adoc: line 1540: section title out of sequence: expected level 3, got level 4
    asciidoctor: WARNING: backuprestore.adoc: line 1549: section title out of sequence: expected level 3, got level 4
    asciidoctor: WARNING: remote_imap.adoc: line 18: section title out of sequence: expected level 2, got level 3
    asciidoctor: WARNING: remote_imap.adoc: line 31: section title out of sequence: expected level 2, got level 3

This was observed when running the following command:

    asciidoctor -b html5 -o admin_guide.html index.adoc

**Comment by ilyusha** [comment]
@gordyt the latest commit fixes these warnings

**Pull request merged**